include/DrawTransform.hpp

```diff
@@ -0,0 +1,19 @@
+#ifndef DRAWTRANSFORM_INCLUDE
+#define DRAWTRANSFORM_INCLUDE
+
+#include <SFML/Graphics/Drawable.hpp>
+#include <SFML/Graphics/RenderTarget.hpp>
+#include <SFML/Graphics/Transformable.hpp>
+#include <SFML/System/Vector2.hpp>
+
+
+// Class that contains Drawable and Transformable without any modification
+// to use at specialization of Listaestatica
+class DrawTransform : public sf::Drawable, public sf::Transformable
+{
+public:
+	DrawTransform();
+	~DrawTransform();
```

Comment:
Se o destrutor não possuir nenhum comportamento especial usar: 
```virtual ~DrawTransform() = default;```
ou
```virtual ~DrawTransform() {}```
Dependendo da sua aplicação.
Leitura recomendada: https://stackoverflow.com/questions/13576055/how-is-default-different-from-for-default-constructor-and-destructor

Reply:
O construtor é padrão mesmo, vou alterar.